src/Stratis.Bitcoin.Features.Wallet.Tests/WalletManagerTest.cs

```diff
@@ -292,14 +281,16 @@ public void CreateWalletWithWalletSetting100UnusedAddressBufferCreates100Address
 
             var walletManager = this.CreateWalletManager(dataFolder, this.Network, "-walletaddressbuffer=100");
 
+            walletManager.Start();
+
             walletManager.CreateWallet("test", "mywallet", "this is my magic passphrase", new Mnemonic(Wordlist.English, WordCount.Eighteen));
 
             HdAccount hdAccount = walletManager.Wallets.Single().AccountsRoot.Single().Accounts.Single();
 
             Assert.Equal(100, hdAccount.ExternalAddresses.Count);
             Assert.Equal(100, hdAccount.InternalAddresses.Count);
         }
-
+        /*
```

Comment:
This file contains a lot tests that are commented out. Can you perhaps just add a comment so that we know why they are not being run?

Reply:
Some tests have been commented out due to either not being relevant or perhaps seen as something that can be fixed in parallel with the testers doing their work. I.e. the fix did not look straightforward at the time and there is no reason for it to hold as back at this time.